en/identity-server/7.0.0/docs/complete-guides/react/next-steps.md

```diff
@@ -50,6 +50,6 @@ npm run dev
 Now that your React application is secured with authentication features integrated, It is time to explore the additional features {{product_name}} offers to make the login flow more diverse and secure.
 
 - [Multi factor authentication](https://is.docs.wso2.com/en/latest/guides/authentication/mfa/){:target="_blank"} 
-- [Passwordless authentication](hhttps://is.docs.wso2.com/en/latest/guides/authentication/passwordless-login/){:target="_blank"} 
+- [Passwordless authentication](https://is.docs.wso2.com/en/latest/guides/authentication/passwordless-login/){:target="_blank"} 
```

Comment:
Shouldn't we take the base path from a config rather than hardcoding

Reply:
+1 fixed with https://github.com/wso2/docs-is/pull/5141/commits/b32df84ad27bbeef6037a183aebc2f68824d5422